aptos-move/framework/aptos-framework/sources/Genesis.move

```diff
@@ -30,6 +30,10 @@ module AptosFramework::Genesis {
         epoch_interval: u64,
         minimum_stake: u64,
         maximum_stake: u64,
+        min_lockup_time_secs: u64,
+        max_lockup_time_secs: u64,
+        post_genesis_validator_set_change_allowed: bool,
```

Comment:
shall we make this variable consistent with the tooling side? `allow_new_validators ` is in the cli PR, but I guess it make sense to say allow change, maybe something like `allow_validator_set_change` ?

Reply:
shorter name please

Reply:
Updated.